static/styles/subscriptions.css

```diff
@@ -483,7 +483,7 @@ form#add_new_subscription {
 }
 
 .subscriptions-container .right {
-    width: calc(50% + 1px);
+    width: calc(50% - -1px);
```

Comment:
This one is too minute a change to even notice but maybe on high res screens can reproduce.
I am attaching a gif. Look at the lower line border of the stream settings bar. It gets shifted by 1px when this setting is fixed. That makes it align with the upper line border.
![calc-5](https://user-images.githubusercontent.com/17237412/36867033-fb5ba9ca-1db9-11e8-89a0-2147219cd07e.gif)